bin/annotate_replays.py

```diff
@@ -0,0 +1,237 @@
+import cv2
+import csv
+import getopt
+import sys
+import torch
+import numpy as np
+import os
+
+# python bin/annotate_video.py -v  no_commit/short_test.mp4 -o short_annotations -f 0
+
+
+def draw_box(image, data, bgr):
+  x_shape, y_shape = image.shape[1], image.shape[0]
+  x1, y1, x2, y2 = int(data[0]*x_shape), int(data[1]*y_shape), int(data[2]*x_shape), int(data[3]*y_shape)
+  cv2.rectangle(image, (x1, y1), (x2, y2), bgr, 1)
+  return image
+
+obj_det_model = torch.hub.load('ultralytics/yolov5', 'custom', 'ai_models/object_detection/trained/object_detect5.pt')  # custom trained model
+obj_det_model.conf = 0.2
+
+start_frame = 0
+end_frame = 0
+video_path = ''
+out_path = ''
+
+try:
+  opts, args = getopt.getopt(sys.argv[1:],"v:o:f:")
+except getopt.GetoptError:
+  print('error')
```

Comment:
Could we give a more descriptive error here? I'm not familiar with getopt. Does the GetoptError object have some sort of error text we can print, or just hardcode some text that says where something's gone wrong?